### PR TITLE
Add support for unique TextFields

### LINKED ISSUE #331: `TextField` is not supported with unique value generation
**Describe the bug**
A unique `TextField` is not treated the same as a unique `CharField` and no new unique value is generated.

**To Reproduce**
1. Create a model with a `TextField` marked `unqiue`.
2. Try to clone a model instance.
3. Observe `IntegrityError` being thrown.

**Expected behavior**
`TextField` should be considered equivalent to `CharField` in terms of unique value generation.

**Desktop (please complete the following information):**
 - Version 2.5.3

### PR DESCRIPTION
**Pull request by diesieben07**
Adds support for `unqiue` on `TextField`, similar to `CharField`.
Fixes #331.

### CODE REVIEW

**Review by jackton1 — APPROVED** [review]

File: model_clone/tests/test_clone_mixin.py [hunk 1]
```diff
@@ -649,3 +650,10 @@ def test_cloning_multiple_instances_with_autocommit_is_valid(self):
                 clone.first_name,
                 r"{}\s[\d]".format(Author.UNIQUE_DUPLICATE_SUFFIX),
             )
+
+    def test_cloning_model_with_unique_text_field(self):
+        product = Product.objects.create(name="Test Product")
+        clone = product.make_clone()
+        self.assertEqual(
+            clone.name, f"{product.name} {Product.UNIQUE_DUPLICATE_SUFFIX} 1"
```
Comment (jackton1) [inline model_clone/tests/test_clone_mixin.py:658]: Seems this line is failing on python2.7 which doesn't support `f-strings`

Screen Shot 2021-05-17 at 1 41 51 PM

**Review by jackton1 — CHANGES_REQUESTED** [review]

**Review by jackton1 — APPROVED** [review]

File: model_clone/tests/test_clone_mixin.py (hunk 1 above)
Comment (diesieben07) [reply model_clone/tests/test_clone_mixin.py]: Apologies, I didn't know you still supported Python 2 (and neither did I know that those were a Python 3 feature).

Comment (jackton1) [reply model_clone/tests/test_clone_mixin.py]: No worries. Thanks

**Review by jackton1 — APPROVED** [review]

**Comment by jackton1** [comment]
@all-contributors please add @diesieben07 for test, bug

**Comment by jackton1** [comment]
@all-contributors please add @diesieben07 for code

**Pull request merged**